.github/workflows/sanity-test.yaml

```
@@ -115,12 +117,28 @@ jobs:
           data=$( echo $resultjson | jq -r '.[0] | .id' )
           echo result=$data >> $GITHUB_OUTPUT
 
-      # list the backup exhange
+      # list all exhange backups
       - name: Backup exchange list test
         run: |
           set -euo pipefail
           ./corso backup list exchange \
-          --hide-progress 2>&1 | tee $TEST_RESULT/backup_exchange_list.txt
+          --hide-progress \
+          2>&1 | tee $TEST_RESULT/backup_exchange_list.txt
+
+          if ! grep -q  ${{ steps.exchange-test.outputs.result }} $TEST_RESULT/backup_exchange_list.txt 
+          then
+              echo "listing of backup was not successfull"
+              exit 1
+          fi
+
+      # list the previous exhange backups
+      - name: Backup exchange list test
```

Is this test case added to test the `--backup` flag? And if it is so can we rename it to- 
 
```suggestion
      - name: List single exchange backup
```
or something similar

---

yep, it's just the added --backup flag